### PR TITLE
Office 2022q1 report

### PR DESCRIPTION
**Pull request by fluffykhv**

### CODE REVIEW

File: 2022q1/office.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Contact: FreeBSD Office team ML <office@FreeBSD.org> +
+Contact: Dima Panov <fluffy@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+The FreeBSD Office team works on a number of office-related software suites and tools such as OpenOffice and LibreOffice.
+
+Work during this quarter was focused on providing the latest stable release of LibreOffice suite and companion apps to all FreeBSD users.
+
+During the 2022Q1 period we are pushed maintenance patches for LibreOffice 7.2 leaf to the quarterly branch and bring the latest, 7.3, releases
```
Comment (pauamma) [inline 2022q1/office.adoc:14]: s/are pushed/pushed/

**Review by pauamma — CHANGES_REQUESTED** [review]
Can you redo this without the upstream merge? See https://github.com/freebsd/freebsd-quarterly/pull/455#issuecomment-1078747188 for the suggested workflow.

**Comment by lwhsu** [comment]
I fixed the grammar and did some updates. A new PR created https://github.com/freebsd/freebsd-quarterly/pull/465 for workflow issue.

**Comment by pauamma** [comment]
Closing in favor of #465 as all changes requested here are present there. Thanks.

**Pull request closed without merging**